Library/Homebrew/dev-cmd/pr-pull.rb

```diff
@@ -108,51 +108,62 @@ def signoff!(path, pr: nil, dry_run: false)
     end
   end
 
-  def determine_bump_subject(old_contents, new_contents, formula_path, reason: nil)
-    formula_path = Pathname(formula_path)
-    formula_name = formula_path.basename.to_s.chomp(".rb")
+  def get_package(tap, subject_name, subject_path, content)
+    if subject_path.dirname == tap.cask_dir
+      begin
+        Cask::CaskLoader.load(content.dup, config: Cask::Config.defaults)
+      rescue Cask::CaskUnavailableError => e
+        ohai e
+        nil
+      end
+    end
```

Comment:
```suggestion
  def get_package(tap, subject_name, subject_path, content)
    if subject_path.dirname == tap.cask_dir
      cask = begin
        Cask::CaskLoader.load(content.dup, config: Cask::Config.defaults)
      rescue Cask::CaskUnavailableError => e
        ohai e
        nil
      end

      return cask
    end
```
Your problem is because the value isn't returned from the function 😅.